src/components/DisbursementInviteMessage/index.tsx

```diff
@@ -0,0 +1,156 @@
+
+  const [selectedOption, setSelectedOption] = useState(radioValue.ORGANIZATION);
+  const [customMessageInput, setCustomMessageInput] = useState(
+    customMessagePlaceholder,
+  );
+
+  useEffect(() => {
+    dispatch(getOrgInfoAction());
+  }, [dispatch]);
+
+  const handleOptionChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (event.target.value === radioValue.ORGANIZATION) {
+      setCustomMessageInput("");
+      setSelectedOption(radioValue.ORGANIZATION);
+    } else {
+      setSelectedOption(radioValue.CUSTOM);
+    }
+  };
+
+  const updateMessage = (updatedDisbursementInviteMessage: string) => {
+    // Updating parent
+    if (onChange) {
+      onChange(updatedDisbursementInviteMessage);
+    }
+  };
+
+  const updateCustomMessageInput = (
+    event: React.ChangeEvent<HTMLTextAreaElement>,
+  ) => {
+    const { value } = event.target;
+
+    setCustomMessageInput(value);
+    updateMessage(value);
+  };
+
+  return (
+    <>
+      <Card>
+        <div className="CardStack__card ReceiverInviteMessage">
+          <div className="CardStack__title">Customize invite</div>
+
+          <div className="Note">
+            You can use the organization default message or customized text to
+            invite your receivers to set up a wallet and receive funds for this
+            disbursement. It contains a link to the appropriate wallet at the
+            end of the message. Please check all values for accuracy.
+          </div>
+
+          {isEditMessage ? (
+            <>
+              <div className="ReceiverInviteMessage__options">
+                <RadioButton
+                  id="msg-std"
+                  name="receiver-message"
+                  label={
+                    <Fragment key="msg-std-label">
+                      {"Default message"}{" "}
+                    </Fragment>
+                  }
+                  fieldSize="xs"
+                  value={radioValue.ORGANIZATION}
+                  onChange={handleOptionChange}
+                  checked={selectedOption === radioValue.ORGANIZATION}
+                />
+                <RadioButton
+                  id="msg-cst"
+                  name="receiver-message"
+                  label="Custom message"
+                  fieldSize="xs"
+                  value={radioValue.CUSTOM}
+                  onChange={handleOptionChange}
+                  checked={selectedOption === radioValue.CUSTOM}
+                />
+              </div>
+              {selectedOption === radioValue.CUSTOM ? (
+                <form className="ReceiverInviteMessage__form">
+                  <Textarea
+                    fieldSize="sm"
+                    id="textarea-custom-input"
+                    rows={5}
+                    value={customMessageInput}
+                    onChange={(event) => {
+                      setCustomMessageInput(event.target.value);
+                      updateCustomMessageInput(event);
+                    }}
```

Comment:
We're calling `setCustomMessageInput` twice (it's included in `updateCustomMessageInput` method). If the custom message is updated only in one place, we could simplify the flow by doing everything directly here. The more functions we add, the higher the chance we'll miss something, so I'd try to avoid that if possible.